src/PyHyperScattering/SST1RSoXSDB.py

```diff
@@ -1025,72 +1044,50 @@ def loadMd(self, run):
-                            if (
-                                type(blval) == tiled.client.array.ArrayClient
-                                or type(blval) == tiled.client.array.DaskArrayClient
-                            ):
-                                blval = blval.read()
-                            md[phs] = blval.mean().round(4)
-                            if blval.var() > 0:
-                                warnings.warn(
-                                    (
-                                        f"While loading {md_secondary_lookup[phs]} to infill"
-                                        f" metadata entry for {phs}, found beginning and end"
-                                        f" values unequal: {baseline[rsoxs]}.  It is possible"
-                                        " something is messed up."
-                                    ),
-                                    stacklevel=2,
-                                )
-                        except (KeyError, HTTPStatusError):
+                        baselineValue = baseline[keyName_Beamline] ## Next, try finding metadata in baseline
+                        if (
+                            type(baselineValue) == tiled.client.array.ArrayClient
+                            or type(baselineValue) == tiled.client.array.DaskArrayClient
+                        ):
+                            baselineValue = baselineValue.read() ## For tiled_client.array data types, need to use .read() to get the values
+                        md[keyName_PHS] = baselineValue.mean()
+                        if baselineValue.var() > 0: ## Might need to increase tolerance, so that it does not throw unnecessary warnings for small variations
                             warnings.warn(
                                 (
-                                    f"Could not find {rsoxs} in either baseline or primary. "
-                                    f" Needed to infill value {phs}.  Setting to None."
+                                    f"While loading {keyName_Beamline} to infill metadata entry for {keyName_PHS}, found beginning and end values unequal: {baselineValue}.  It is possible something is messed up."
                                 ),
                                 stacklevel=2,
                             )
-                            md[phs] = None
-        md["epoch"] = md["meas_time"].timestamp()
+                    except (KeyError, HTTPStatusError): md[keyName_PHS] = None
+            if md[keyName_PHS] is None:
+                warnings.warn(
+                    (
+                        f"Could not find {keyNames_Beamline} in either baseline or primary. Setting {keyName_PHS} to None.  Can be entered manually."
+                    ),
+                    stacklevel=2,
+                )
+                        
+        md["epoch"] = md["meas_time"].timestamp() ## Intended to classify all data that belongs to a single scan
```

Comment:
```suggestion
        md["epoch"] = md["meas_time"].timestamp() # Epoch = the time the entire run started, used for multi-scan stacking
```

A documentation page that covered these words and their function would be a very welcome improvement...

Reply:
Sure, though I'm not sure completely understand the purpose of epoch yet.  Could you provide an example?  When is it beneficial to stack by just the start time rather than by the entire list of timestamps in the scan?

Reply:
If I want to stack four scans that don't have distinct other features, epoch is a last-resort differentiator.  `scanid` should function similarly.  Also, if I want to stack the same scan at different wall-clock times to look at like beamline stability.  The epoch is the property of the entire scan, so I can do e=270 at three epochs, where if I didn't have the epoch axis I'd have to do some complicated co-slicing of timestamp.

Reply:
One (hopefully) final question, is there a reason we're calling it `epoch` rather than something more specific like `start_time`?